### PR TITLE
Searchlist video

### PR DESCRIPTION
**Pull request by santhoshpemmaka**
In this PR Request add a search feature, signup feature, and toast to the video library.

## Things to check

* use this link to see files that need to be reviewed ▶️ [Review me only](https://github.com/santhoshpemmaka/NamasteVideo/pull/4/files?file-filters%5B%5D=.jsx&show-viewed-files=true)

* Deployment link:- [Video Library](https://namaste-video-library.netlify.app/)

### CODE REVIEW

**Review by iRohitGaur — APPROVED** [review]
UI looks good.

Nice work! Good to merge 👍🏻

File: src/App.jsx [hunk 1]
```diff
@@ -14,6 +15,7 @@ import {
 	HistoryVideoListing,
 	PlaylistVideoListing,
 	ProfileListing,
+	ErrorPage,
 } from "./components";
 import {useData} from "./context/VideoContext";
 import {getVideosServer} from "./utils/server-request";
```
Comment (Shubhamkr2610) [inline src/App.jsx:21]: appreciation: good way of importing files

**Review by Shubhamkr2610 — APPROVED** [review]
Good naming of classes . Looks everything good to me , code is clean and readable and  easy to understand . you can go for merge

**Comment by santhoshpemmaka** [comment]
Merge to the Home Page branch

File: src/App.jsx (hunk 1 above)
Comment (santhoshpemmaka) [reply src/App.jsx:21]: thnks

**Pull request merged**